tests/inputs/csv_elements/project_data.csv

```diff
@@ -6,3 +6,4 @@ project_id,str,1
 project_name,str,Borg Havn
 scenario_id,str,1
 scenario_name,str,Warehouse 14
+scenario_description,str,"Main features: Demand of harbour and shore power. Connection to National grid (0.54 NOK/kWh electricity price, 60 NOK/kW monthly peak demand pricing, 0.54 NOK/kWh*0.8 feedin tariff. PV system of 300 kWp. Objective: Optimize battery capacity. Outcomes: Capacity battery and utilization profile. Consumption from national grid. Feed-in into national grid. Outlook: With this, it can be compared whether or not the battery reduced the peak demand expenditures. Later, the simulation results should be compared to the consumption from national grid without batteries."
```

Comment:
```suggestion
scenario_description,str,"Main features: Demand of harbour and shore power. Connection to National grid (0.54 NOK/kWh electricity price, 60 NOK/kW monthly peak demand pricing, 0.54 NOK/kWh*0.8 feedin tariff. PV system of 300 kWp. Objective: Optimize battery capacity. Outcomes: Capacity battery and utilization profile. Consumption from national grid. Feed-in into national grid. Outlook: With this, it can be assessed whether or not the battery reduced the peak demand expenditures. Later, the simulation results should be compared to the consumption from national grid without batteries."
```